burner-ui/src/Pages/HomePage/HomePage.tsx

```diff
@@ -31,13 +31,25 @@ const StyledPage = styled(Page)`
   padding-bottom: 110px;
 `;
 
-const HomeButton: React.FC<HomeButtonProps> = ({ path, title, classes }) => (
-  <li className={classes.buttonContainer}>
-    <Button to={path} className={classes.homeButton}>
-      {title}
-    </Button>
-  </li>
-);
+const SubHeading = styled(L2)`
```

Comment:
Not against this, just curious for your thinking behind it 😃

Reply:
I needed to add some CSS to the existing L2 element, how would you have done it?

Reply:
I have a personal "rule" that I follow, where I prefer to use my containers as the elements that dictate the layout, rather than individual elements. This is mostly just for consistency, and to establish a division between layout and content.

In this case, I would have set the container of the `Recent Activity` and `View All` buttons to  `justify-content: space-between` and added `margin` to it to improve its spacing.

By removing the `margin` from the `Recent Activity` element you actually reduce the height of the `View All` button at the same time, which is set to fill whatever height its container has.

Reply:
@as-dr Ah thanks, that makes a lot of sense.

Feel free to clean this up on develop if you want.